bee-network/examples/common/args.rs

```diff
@@ -0,0 +1,39 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use super::config::ExampleConfig;
+
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+#[structopt(name = "pingpong", about = "bee-network example")]
+pub struct Args {
+    #[structopt(short = "b", long = "bind")]
+    bind_address: String,
+
+    #[structopt(short = "p", long = "peers")]
+    peer_addresses: Vec<String>,
+
+    #[structopt(short = "m", long = "msg")]
+    message: String,
+}
+
+impl Args {
+    pub fn into_config(self) -> ExampleConfig {
```

Comment:
Maybe `impl From<Args> for ExampleConfig`?

Reply:
sure, can do that :+1: